batch/batch/log_store.py

```diff
@@ -8,15 +8,16 @@
 
 
 class LogStore:
-    def __init__(self, bucket_name, instance_id, blocking_pool, *, project=None, credentials=None):
+    def __init__(self, bucket_name, worker_logs_bucket_name, instance_id, blocking_pool, *, project=None, credentials=None):
         self.bucket_name = bucket_name
+        self.worker_logs_bucket_name = worker_logs_bucket_name
         self.instance_id = instance_id
         self.log_root = f'gs://{bucket_name}/batch/logs/{instance_id}'
```

Comment:
Can we change this to `batch_log_root` or something otherwise descriptive of its now limited purpose?

Reply:
Good call, will do.